### PR TITLE
fix: oras commands to not hide error output

### PR DESCRIPTION
**Pull request by mmalina**
I believe the redirection of standard error output was done by mistake, probably assuming that it might polute the variable we're assigning to. But that's not how it works
 - stderr is not used for the assignment.

And recently a user saw a failure of the oras command, but there was no error output, so it was impossible to figure out what happened.

See here: https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1721633939965029

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]

**Comment by mmalina** [comment]
@johnbieren I tried to fix the checkton warnings. There are two. In both cases we want to ignore them - the oras args shouldn't be wrapped in `""`. You ignore stuff with comments like `# shellcheck disable=SC2116` on the line above. That's easy to do with the first one, but the second one is tricky and I couldn't figure out how to do it - because the line is split in two using `\` and you can't add a comment row in between the two parts and if you add it above the first one, it doesn't work.

Long story short, I'd like to ignore these warnings in this PR and leave this for later when we fix all the tasks.

**Review by johnbieren — APPROVED** [review]
I am fine with ignoring checkton until we fix it in all tasks
